src/main/resources/database/data.sql

```diff
@@ -36,13 +36,11 @@ VALUES ('Завхоз', false);
 INSERT INTO `user` (u_login, u_password, position_id)
 VALUES ('admin', '$2a$10$yKj28aFTziVYvmegHoSLy.WZVPJINyt8xCKZDhSPfxu2j7dtAFtRe', 1);
 INSERT INTO `user` (u_login, u_password, position_id)
-VALUES ('login2', '1', 2);
+VALUES ('user1', '$2a$10$sUxH.CoUiCE9JRY9suExP.c6m9VYyDORkWZn4ciq8im6RzXMPcOfK', 2);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([plsql:S1192](https://sonarcloud.io/organizations/final-project/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=180&issues=AWb5WfX4W3Y69bz5BsNk&open=AWb5WfX4W3Y69bz5BsNk)

[ISSUE_KEY]: AWb5WfX4W3Y69bz5BsNk